beaglebone-play.coffee

```diff
@@ -0,0 +1,52 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
```

Comment:
This entire commit needs to be removed, we decided to base the device type on the upstream BSP machine which is called `beagleplay`